### DISCUSSION TITLE
Cannot configure docker container

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [Taxel/PlexTraktSync#970] [Support] [answered]
The documentation isn't exactly clear about how to configure plextraktsync when running the cocker container...

I tried to [run the docker image in interactive mode as suggested in an old issue](https://github.com/Taxel/PlexTraktSync/issues/68), but the container ends with an error

```
$ docker start -ia plextraktsync
INFO     PlexTraktSync [0.20.4]
Please enter your Plex username or e-mail: Error: Error running sync command:
```

The solution was as explained in [Container stops/restart after creation - If no user configured](https://github.com/Taxel/PlexTraktSync/issues/767#issuecomment-1026309892) to create the container manually instead of through docker compose.

This should be added to the documentation

**Comment by glensc** [comment]
So send the PR to add to documentation

**Comment by Th3S4mur41** [comment, marked as answer]
PR created 😉
https://github.com/Taxel/PlexTraktSync/pull/974
